### ISSUE TITLE
Satisfies return false if browser is not mentionned

### ISSUE THREAD

**Issue by Clebiez** [lancedikson/bowser#407]
Here my satisfies object.

```
const ua = window.navigator.userAgent;
const browser = Bowser.getParser(ua);
export const isBrowserSupported = browser.satisfies({
  windows: {
    'internet explorer': '>10',
  },
  chrome: '>40',
  firefox: '>40',
  safari: '>6',
  opera: '>30'
});
```

If user come with Edge version, satisfies will return false.

In my case I used bowser v1 to identify user with old browsers and show them a page "Ugprade your browser" instead of access to my application.
With this method which return false by default, I must pass all browsers and maintain this list.
It's very complicated so maybe can we add an option parameter like "strict: false" in order to return true if browser is not passed in the list.

**Comment by michasik** [comment]
I totally agree.
Now the workaround is to check if the browser name is present in the object passed to `browser.satisfies()` as mentioned above.

**Comment by cagline** [comment]
Yes totally agreed, it should be restrict only browsers which are mention.
In this cause user need to give all the browsers in the world.

**Comment by ybrodsky** [comment]
Just change the way you check.
`isBrowserSupported` will be false when the browser is not supported and it will be true when its supported (according to what you defined). And if the browser its not defined in the list, it returns undefined. So

```
const isBrowserSupported = browser.satisfies({
  windows: {
    'internet explorer': '>10',
  },
});

// when browser is chrome isBrowserSupported=undefined
// when browser is ie 11, isBrowserSupported=true
// when browser is ie 9, isBrowserSupported=false

if (isBrowserSupported === false) {
  // browser not supported
}

// else is supported
```
